addons/binding/org.openhab.binding.nikohomecontrol/src/main/java/org/openhab/binding/nikohomecontrol/internal/NikoHomeControlHandlerFactory.java

```diff
@@ -41,19 +47,28 @@
  */
 
 @Component(service = ThingHandlerFactory.class, configurationPid = "binding.nikohomecontrol")
+@NonNullByDefault
 public class NikoHomeControlHandlerFactory extends BaseThingHandlerFactory {
 
-    private Map<ThingUID, ServiceRegistration<?>> discoveryServiceRegs = new HashMap<>();
+    @NonNullByDefault({})
+    private NetworkAddressService networkAddressService;
```

Comment:
We use the convention to put the null annotations for fields on the same line.
```suggestion
    private @NonNullByDefault({}) NetworkAddressService networkAddressService;
```

This comment applies to all occurrences in the code

Reply:
Done.